MyHashSet.java

```diff
@@ -0,0 +1,79 @@
+
+public class MyHashSet {
+    /*
+     * Time Complexity: O(1) for adding, removing and searching an element in HashSet
+     * Space Complexity: A little sketchy on this. I feel this is O(n) as the fixed size of array for buckets (primary
+     * array is given). Then O(log(n)) for secondary array in the best case. Can this be O(n(log(n)) considering in the
```

Comment:
The Worse case complexity can be states as `O(k^2)` here where k is the size of the bucket.